curriculum/challenges/english/10-coding-interview-prep/rosetta-code/averages-root-mean-square.md

```diff
@@ -8,11 +8,9 @@ dashedName: averagesroot-mean-square
 
 # --description--
 
-Compute the [Root mean square](<https://en.wikipedia.org/wiki/Root mean square> "wp: Root mean square") of the numbers 1 through 10 inclusive.
+Compute the Root mean square of the numbers 1 through 10 inclusive.
```

Comment:
```suggestion
Compute the Root Mean Square (RMS) of the numbers 1 through 10 inclusive.
```
I think its standard to define abbreviations like this in line before using them..